content/cheatsheet.md

```diff
@@ -116,6 +130,57 @@ To make an image appear on the next line in a list:
 1. Add 2 spaces at the end of the line and put the image on the next line with a tab indentation.  
     ![Redis Enterrpise Cluster]( /images/rs/rp_stack.png )
 
+### Image shortcode
```

Comment:
Is there a functional difference between this and ![]() ?

Reply:
Image shortcode (ie. {{% image..}}) can only display images from the site itself, ie. images located in _static_ directory (or its subdirectories), whereas the other way (ie. ![Redis Enterrpise Cluster]( /images/rs/rp_stack.png )) can display both those images and external images from other domains.

Therefore I think that the ![Redis Enterrpise Cluster]( /images/rs/rp_stack.png ) option is sufficient as it covers both local and external images.

(similar to code shortcode, this was documented to have all working shortcodes in one place and can be removed from the cheatsheet).

Reply:
👍